File: poo-06/src/Aluno.java
public class Aluno {
    private String nome;
    private String ra;
    private int idade;
    private String genero;
    private boolean status;

    public Aluno(String nome, String ra, int idade, String genero, boolean status) {
        this.nome = nome;
        this.ra = ra;
        this.idade = idade;
        this.genero = genero;
        this.status = status;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }
    public String getNome() {
        return nome;
    }
}

File: pw-27-05/src/models/entity/aluno.php
<?php
    namespace src\models\entity;

    class aluno {
        private $id;
        private $nome;
        private $genero;

        public function __construct($nome, $genero) {
            $this -> nome = $nome;
            $this -> genero = $genero;
        }
        
        public function getId() {
            return $this -> id;
        }
        
        public function getNome() {
            return $this -> nome;
        }
        
        public function getGenero() {
            return $this -> genero;
        }

        public function setId($id) {
            $this->id = $id;
        }

        public function setNome($nome) {
            $this -> nome = $nome;
        }

        public function setGenero($genero) {
            $this -> genero = $genero;
        }
    }
?>